pkg/registry/core/service/storage/rest_test.go

```diff
@@ -673,22 +774,6 @@ func TestServiceRegistryResourceLocation(t *testing.T) {
 					Ports:     []api.EndpointPort{{Name: "", Port: 80}, {Name: "p", Port: 93}},
 				}},
 			},
-			{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "foo",
```

Comment:
this was a duplicate name/namespace? what was this doing before?

Reply:
the first one in the list was getting chosen it looks like by name.  someone added the second name, the test passed, went on their way.  will dig in a bit more.